views/Login/index.tsx

```diff
@@ -1,51 +1,49 @@
-import { useLayoutEffect } from 'react'
+import { useLayoutEffect as useEffect } from 'react'
 import { NextPage } from 'next'
-import { useRouter } from 'next/router'
 import { useLazyQuery } from '@apollo/client'
-import { toast } from 'react-toastify'
 import { GET_USER_SESSION } from '@/services/GraphQL/queries/users'
 import { LoginInput } from '@/services/GraphQL/types/users'
-import UseError from '@/hooks/useError'
+import useLogger from '@/hooks/useLogger'
+import useMagicLink from '@/hooks/useMagicLink'
 import LoginLayout from './Layout'
 
 const Login: NextPage = () => {
-  const router = useRouter()
   const [checkUserSession, { loading, data }] = useLazyQuery(GET_USER_SESSION)
-  const [createError] = UseError()
+  const { log, warn, error } = useLogger()
+  const magicLink = useMagicLink()
 
-  useLayoutEffect(() => {
-    if (data?.users?.length === 1) {
-      window.sessionStorage.setItem('userId', data.users[0].id)
-      window.sessionStorage.setItem('user', JSON.stringify(data.users[0]))
-      router.push('/eventos/')
-    }
+  useEffect(() => {
+    if (!data) return
     if (data?.users?.length === 0) {
-      toast('Usuario o contraseña incorrectos', { type: 'error' })
+      warn(
+        'Login:useEffect',
+        'No se encontró ningún usuario en base de datos, Usuario o contraseña incorrectos...',
+        'INPUT',
+      )
+    } else {
+      if (data?.users?.length > 1) {
+        warn(
+          'Login:useEffect',
+          'Se encontraron varios usuarios en base de datos, usando el primer email. Informe al servicio técnico del incidente...',
+          'INPUT',
+        )
+      }
```

Comment:
This case can occurs if a user have the same member code and the same password

Reply:
But shouldn't be possible since the database should have the username as a unique primary key field

Reply:
Ah, I left the case if something is done wrong in DB. I think is like a safety net. Will change the type to "OTHER" and add a note about uniqueness check-in DB for codeMember.